crates/y-sweet-worker/src/config.rs

```diff
@@ -10,6 +10,7 @@ const CHECKPOINT_FREQ_SECONDS: &str = "CHECKPOINT_FREQ_SECONDS";
 const S3_ACCESS_KEY_ID: &str = "AWS_ACCESS_KEY_ID";
 const S3_SECRET_ACCESS_KEY: &str = "AWS_SECRET_ACCESS_KEY";
 const S3_REGION: &str = "AWS_REGION";
```

Comment:
AWS uses `AWS_DEFAULT_REGION`.

Reply:
It uses both, which one do you think is preferable?

Reply:
Good point, let's use `AWS_REGION`.